biogrid/libs/biogrid-simulator/src/lib/biogrid/biogrid-class.ts

```diff
@@ -89,18 +100,32 @@ export class Biogrid implements Grid {
     return this.state.getJsonGraph();
   }
 
-  private createBatteries(positions: ItemPosition[], gridItemName: string): Battery[] {
-    const batteryResistance = gridItemName === GRID_ITEM_NAMES.LARGE_BATTERY
-      ? RESISTANCE.LARGE_BATTERY
-      : RESISTANCE.SMALL_BATTERY;
-    const maxCapacity = gridItemName === GRID_ITEM_NAMES.LARGE_BATTERY
-      ? LARGE_BATTERY.MAX_CAPACITY
-      : SMALL_BATTERY.MAX_CAPACITY;
-    const initEnergy = gridItemName === GRID_ITEM_NAMES.LARGE_BATTERY
-      ? LARGE_BATTERY.DEFAULT_START_ENERGY
-      : SMALL_BATTERY.DEFAULT_START_ENERGY;
+  private createBatteries(
+    positions: ItemPosition[],
+    gridItemName: string
+  ): Battery[] {
+    const batteryResistance =
+      gridItemName ===  bioconstants.GRID_ITEM_NAMES.LARGE_BATTERY
+        ? bioconstants.RESISTANCE.LARGE_BATTERY
+        : bioconstants.RESISTANCE.SMALL_BATTERY;
+    const maxCapacity =
+      gridItemName === bioconstants.GRID_ITEM_NAMES.LARGE_BATTERY
+        ? bioconstants.LARGE_BATTERY.MAX_CAPACITY
+        : bioconstants.SMALL_BATTERY.MAX_CAPACITY;
+    const initEnergy =
+      gridItemName === bioconstants.GRID_ITEM_NAMES.LARGE_BATTERY
+        ? bioconstants.LARGE_BATTERY.DEFAULT_START_ENERGY
+        : bioconstants.SMALL_BATTERY.DEFAULT_START_ENERGY;
     return positions.map(
-      (position, index) => new BioBattery(position.x, position.y, `${gridItemName}-${index}`, batteryResistance, initEnergy, maxCapacity)
+      (position, index) =>
+        new BioBattery(
+          position.x,
+          position.y,
+          `${gridItemName}-${index}`,
+          batteryResistance,
+          initEnergy,
+          maxCapacity
```

Comment:
I believe @cwlarson-google mentioned this in another pr - for constructors that take a lot of parameters, consider using a single object parameter instead

Reply:
Fixed!